src/editor-mathfield/pointer-input.ts

```diff
@@ -19,7 +19,12 @@ export function onPointerDown(
 ): void {
   //Reset the atom bounds cache
   mathfield._atomBoundsCache = new Map<string, Rect>();
-
+  if (
+    (evt.target as HTMLElement).tagName === 'MATH-FIELD' &&
```

Comment:
the case of the tag is not guaranteed to be uppercase (it's lowercase in some browsers). So you need to do a case-insensitive comparison, i.e. `tagName.toLowerCase() === 'math-field'`